backend/views/syson-diagram-general-view/src/main/java/org/eclipse/syson/diagram/general/view/services/GeneralViewNodeToolSectionSwitch.java

```diff
@@ -185,4 +195,23 @@ private NodeTool addExistingNestedPartsTool() {
                 .body(addExistingelements.build())
                 .build();
     }
+
+    private void createToolsForCompartmentItems(Element object) {
+        List<NodeTool> compartmentNodeTools = new ArrayList<>();
+        GeneralViewDiagramDescriptionProvider.COMPARTMENTS_WITH_LIST_ITEMS.forEach((compartmentEClass, listItems) -> {
+            if (compartmentEClass.equals(object.eClass())) {
+                listItems.forEach(eReference -> {
+                    CompartmentNodeToolProvider provider = new CompartmentNodeToolProvider(eReference.getEType());
+                    compartmentNodeTools.add(provider.create(null));
+                });
+            }
+        });
+        Optional<NodeToolSection> createToolSection = nodeToolSections.stream().filter(toolSection -> toolSection.getName().equals("Create")).findFirst();
```

Comment:
Could you add `this.` as much as possible when using class variable please?

Reply:
done